webpack/assets/javascripts/react_app/components/FactCharts/__test__/FactChart.test.js

```diff
@@ -22,12 +22,4 @@ describe('factCharts', () => {
     );
     expect(toJson(wrapper)).toMatchSnapshot();
   });
-
-  it('should open modal after click', () => {
```

Comment:
This test doesn't test anything, because `FactChart` component doesn't receive `onClick` at all.
This test is broken after upgrading redux.